packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -30,25 +42,21 @@ export function menuItemTemplate(
             ${when(
                 x => x.role === MenuItemRole.menuitemcheckbox,
                 html<FASTMenuItem>`
-                    <div part="input-container" class="input-container">
-                        <span part="checkbox" class="checkbox">
-                            <slot name="checkbox-indicator">
-                                ${options.checkboxIndicator || ""}
-                            </slot>
-                        </span>
-                    </div>
+                    <span part="checkbox-indicator" class="checkbox-indicator">
+                        <slot name="checkbox-indicator">
+                            ${options.checkboxIndicator || checkboxIndicator}
+                        </slot>
+                    </span>
                 `
             )}
             ${when(
                 x => x.role === MenuItemRole.menuitemradio,
                 html<FASTMenuItem>`
-                    <div part="input-container" class="input-container">
-                        <span part="radio" class="radio">
-                            <slot name="radio-indicator">
-                                ${options.radioIndicator || ""}
-                            </slot>
-                        </span>
-                    </div>
+                    <span part="radio-indicator" class="radio-indicator">
+                        <slot name="radio-indicator">
+                            ${options.radioIndicator || radioIndicator}
+                        </slot>
+                    </span>
                 `
             )}
         </div>
```

Comment:
I think this should be removed as well, yes?

Reply:
I believe this applies to `radioIndicator` which was a file-level default template which has been removed. Resolving this as "outdated".